### ISSUE TITLE
Standardize how we describe what astropy.time is/enables

### ISSUE THREAD

**Issue by ayshih** [sunpy/sunpy-1.0-paper#99]
I'm annoyed by how the paper describes what `astropy.time` is/enables.

### Section 2:
```
...a high precision time format...
```
The high-precision capability of `astropy.time` is arguably the least useful benefit for SunPy.  I think everything else on the bulleted list in the SEP (https://github.com/sunpy/sunpy-SEP/blob/master/SEP-0008.md) ranks above it.  I also think that "format" here sounds like we're just writing the time with more digits after the decimal.

### Section 3:
```
...a modern astronomical time system.
```
Putting aside the issue that we're implying that `datetime` is not "modern", this reads as if we are building around a specific "astronomical time system" (e.g., a time scale such as TAI or TDB) rather than something more general.

### Section 8 (the actual time discussion):
```
...a modern scientific time format...
```
I don't like "modern" or "format".  Of course, this sentence is followed up by actual explanation, so the word choice here matters less.

### Bottom line
Can we instead call it "a scientific time class"?  Discuss.

**Comment by hayesla** [comment]
Comments from meeting 

- leap seconds
- cross-disciplinary
- increased precision (probably too much of case - do we really need it for solar)
- handle other timescales - but need supporting information on why this is needed for solar physics
- `datetime` does not support TAI
- need to focus on the fact its a _modern_ astronomical time system, but what does modern mean? need better word

I'll give this a go sure 🕐

**Comment by hayesla** [comment]
light travel times

**Comment by hayesla** [comment]
are we happy to close this issue now?

**Comment by nabobalis** [comment]
We merged the PR. I think it is safe to close this! Thanks @hayesla
